docs/reuse/apps/app-collection-option-2.md

```diff
@@ -0,0 +1,22 @@
+To set up collection and install the app, do the following:
```

Comment:
Should we say "To setup source in existing collector and install, do the following"